client/student/app/lecture/comment.form.directive.js

```diff
@@ -38,10 +38,24 @@
             commentForm.submitted = true;
                         
             if(commentForm.comment.length > commentForm.maxLength){
+            lectureFactory.submitComment(commentForm.comment, function(response) {
+                console.log('Success');
+                commentForm.feedbackMessage = 'Comment submitted!';
+                commentForm.feedbackType = 'alert-success';
+                commentForm.feedbackIcon = 'glyphicon-ok-sign'; 
+                
+                var comment = {
+                    id: response.id,
+                    content: commentForm.comment
+                };
+                
+                commentForm.list.push(comment);
+            }, function() {
```

Comment:
I think you did something weird in the merge. This should not be here. I think you can remove all of this. You can also remove the `});` added below.